heft-plugins/heft-api-extractor-plugin/src/ApiExtractorPlugin.ts

```diff
@@ -155,18 +158,9 @@ export default class ApiExtractorPlugin implements IHeftTaskPlugin {
     taskSession: IHeftTaskSession,
     heftConfiguration: HeftConfiguration
   ): Promise<IApiExtractorTaskConfiguration | undefined> {
-    if (!this._apiExtractorTaskConfigurationFileLoader) {
-      this._apiExtractorTaskConfigurationFileLoader =
-        new ProjectConfigurationFile<IApiExtractorTaskConfiguration>({
-          projectRelativeFilePath: TASK_CONFIG_RELATIVE_PATH,
-          jsonSchemaObject: apiExtractorConfigSchema
-        });
-    }
-
-    return await this._apiExtractorTaskConfigurationFileLoader.tryLoadConfigurationFileForProjectAsync(
-      taskSession.logger.terminal,
-      heftConfiguration.buildFolderPath,
-      heftConfiguration.rigConfig
+    return heftConfiguration.tryLoadProjectConfigurationFileAsync(
```

Comment:
The `_getApiExtractorTaskConfigurationAsync` function is only used in one place. Just inline `heftConfiguration.tryLoadProjectConfigurationFileAsync` on line 174/175?